assets/css/reviewNotice.css

```diff
@@ -0,0 +1,68 @@
+.wemail-review-notice-flex-container {
```

Comment:
The [assets/css/reviewNotice.css](https://github.com/weMail/wemail/pull/296/files#diff-f611fb292f66925d313271bce038e9563ce7f9689f46b9ef51b97ddb90c6cd37) file should `review-notice.css`